Formula/postgrest.rb

```diff
@@ -18,15 +14,16 @@ class Postgrest < Formula
   end
 
   depends_on "cabal-install" => :build
-  depends_on "ghc@8.6" => :build
+  depends_on "ghc" => :build
   depends_on "postgresql"
 
   def install
-    install_cabal_package :using => ["happy"]
+    system "cabal", "v2-update"
+    system "cabal", "v2-install", *std_cabal_v2_args
   end
 
   test do
-    return if ENV["CI"]
+    # return if ENV["CI"]
```

Comment:
Let's delete this line then?

Reply:
Either that or put it back.

It was deliberately done in #53371. I just did not feel comfortable shipping a major version upgrade without properly testing it on all three macOS versions.

The whole avoiding-CI stuff is not really nice to have :(

Reply:
Ah, I hoped it magically get fixed for GitHub action runners

Reply:
The problem was on the underlying machines. It takes a few weeks to appear and at that point the machine needs a restart to fix the issue.

I still don't think #52004 is the proper solution - it could be disasterous when PostgreSQL 13 comes out and the dependents don't get tested (can't just rely on `brew linkage` for PostgreSQL) - but it is what it is.